.vscode/launch.json

```diff
@@ -11,7 +11,9 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "internalConsoleOptions": "openOnSessionStart",
-            "args": ["--extensionDevelopmentPath=${workspaceFolder}"]
+            "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
+            "preLaunchTask": "${defaultBuildTask}"
```

Comment:
I'd rather not do this. I run the watch script so it recompiles automatically on changes. There's no reason to have it compile again when the debugger is started. Watch can't be used as a pre launch task because it never exits.

Reply:
So you run the watch script manually when you want to compile?

Reply:
Do you run the watch script manually after you make changes?

Reply:
Never mind, I get it. Run
```
tsc -watch -p ./tsconfig.watch.json
```
in a shell and it just recompiles every time you save a file.

Reply:
There's also an npm script in package.json so you don't have to type all that out. "npm run watch" will do it. The build tasks will find those, too, so ctrl+shift+b will bring up a list of build tasks and "npm: watch" will be one of those.

Reply:
Using that now, thanks.